CHANGELOG.md

```diff
@@ -56,6 +56,7 @@
   - Accept both string and number for float64.
   - Accept both string and number for int32/uint32.
   - Read uint64 numbers without converting from int64.
+- Rename leftover `label`s to `attribute`s in `loggingexporter` (#5977)
```

Comment:
probably don't need a changelog entry.  One of the approvers will add the Skip Changelog tag so the build passes.